--- a/Marlin/src/lcd/extui/dgus_reloaded/definition/DGUS_VPList.cpp
+++ b/Marlin/src/lcd/extui/dgus_reloaded/definition/DGUS_VPList.cpp
@@ -205,14 +205,14 @@
 
   VP_HELPER_TX_AUTO(DGUS_Addr::TEMP_Current_Bed,
                     &thermalManager.temp_bed.celsius,
-                    &DGUSTxHandler::ExtraToInteger<float>),
+                    (&DGUSTxHandler::ExtraToFixedPoint<float, 1>)),
   VP_HELPER_TX_AUTO(DGUS_Addr::TEMP_Target_Bed,
                     &thermalManager.temp_bed.target,
                     &DGUSTxHandler::ExtraToInteger<int16_t>),
   VP_HELPER_TX(DGUS_Addr::TEMP_Max_Bed, &DGUSTxHandler::TempMax),
   VP_HELPER_TX_AUTO(DGUS_Addr::TEMP_Current_H0,
                     &thermalManager.temp_hotend[ExtUI::heater_t::H0].celsius,
-                    &DGUSTxHandler::ExtraToInteger<float>),
+                    (&DGUSTxHandler::ExtraToFixedPoint<float, 1>)),
   VP_HELPER_TX_AUTO(DGUS_Addr::TEMP_Target_H0,
                     &thermalManager.temp_hotend[ExtUI::heater_t::H0].target,
                     &DGUSTxHandler::ExtraToInteger<int16_t>),
@@ -220,7 +220,7 @@
   #if HAS_MULTI_HOTEND
     VP_HELPER_TX_AUTO(DGUS_Addr::TEMP_Current_H1,
                       &thermalManager.temp_hotend[ExtUI::heater_t::H1].celsius,
-                      &DGUSTxHandler::ExtraToInteger<float>),
+                      (&DGUSTxHandler::ExtraToFixedPoint<float, 1>)),
     VP_HELPER_TX_AUTO(DGUS_Addr::TEMP_Target_H1,
                       &thermalManager.temp_hotend[ExtUI::heater_t::H1].target,
                       &DGUSTxHandler::ExtraToInteger<int16_t>),
